controller.java

```diff
@@ -1,14 +1,10 @@
 
 public class controller {
     public static void main(String[] args) {
-        // Declare two numbers
-        int num1 = 10;
-        int num2 = 20;
-
         // Perform addition
-        int sum = math.add(num1, num2);
+        int sum = math.add(args[1], args[2]);
```

Comment:
change variable name to summation
```suggestion
        int summation = math.add(args[1], args[2]);
```